### PR TITLE
Add pageViewId and cmpInitTimeUtc to PubData

### PR DESCRIPTION
**Pull request by janua**
## What does this change?

This updates the `PubData` type to accept `pageViewId` and `cmpInitTimeUtc`.

We will use `pageViewId` to resolve the `browserId` for the consent signal, and phase out having `browserId` in this type.

We have already been relying in the server side timestamp from Sourcepoint when joining (which in fairness seems pretty accurate without much drift from most clients); however this still should not be relied upon IMO; so I have taken this opportunity to add `cmpInitTimeUtc`.

Maybe not the best name for `cmpInitTimeUtc`; was thinking perhaps of `cmpInitTimestampMillisUtc` as it is an Epoch timestamp in milliseconds.

To @sndrs point around `PubData` prescribing the keys within the object; I haven't changed this to a general object for now as I don't think there will be much movement in this type after this; if we find we are changing it a lot then we should move it to a more general type that accepts any keys.

## How can we measure success?

We can monitor the presense of these new keys in the Sourcepoint logs.

@guardian/commercial-dev

### CODE REVIEW

File: src/types/index.ts [hunk 1]
```diff
@@ -7,6 +7,8 @@ export interface ConsentState {
 }
 export interface PubData {
 	browserId?: string;
+	pageViewId?: string;
+	cmpInitTimeUtc?: number;
```
Comment (sndrs) [inline src/types/index.ts:11]: is there a more accurate type you could use here?

Comment (janua) [reply src/types/index.ts:11]: @sndrs What were you imaginging? Such as a DateTime? It's essentially a long

Comment (sndrs) [reply src/types/index.ts:11]: yeah true, i guess it's just an infinite number of milliseconds?

File: README.md [hunk 2]
```diff
@@ -82,7 +82,14 @@ Pass additional parameters for for reporting. Optional.
 #### Example
 
 ```js
-cmp.init({ pubData: { browserId: 'gow59fnwohwmshz' }, isInUsa: false });
+cmp.init({
+    pubData: {
+        browserId: 'gow59fnwohwmshz',
+        pageViewId: 'jkao3u2kcbaqk',
+        cmpInitTimeUtc: 1601511014537
```
Comment (sndrs) [inline README.md:89]: where would this number come from in practice? `new Date().getTime()`?

File: README.md [hunk 3]
```diff
@@ -82,7 +82,14 @@ Pass additional parameters for for reporting. Optional.
 #### Example
 
 ```js
-cmp.init({ pubData: { browserId: 'gow59fnwohwmshz' }, isInUsa: false });
+cmp.init({
+    pubData: {
+        browserId: 'gow59fnwohwmshz',
```
Comment (sndrs) [inline README.md:87]: it would be useful to add a description of where these values could be acquired

Comment (janua) [reply README.md:92]: @sndrs Description added

**Review by sndrs — APPROVED** [review]

**Pull request merged**